Converter/igtlioVideoConverter.cxx

```diff
@@ -189,6 +196,79 @@ int igtlioVideoConverter::toIGTL(const HeaderData& header, const ContentData& so
     return 0;
     }
 
+  double *spacing;       // spacing (mm/pixel)
+  int   isize[3];          // image dimension
+
+  scalarType = frameImage->GetScalarType();
+  ncomp = frameImage->GetNumberOfScalarComponents();
+  frameImage->GetDimensions(isize);
+  spacing = frameImage->GetSpacing();
+
+  igtl::Matrix4x4 matrix; // Image origin and orientation matrix
```

Comment:
This code block looks familiar. Is it a copy-paste from somewhere else? If so, could we extract it to a static function in a common class?

Reply:
Yup, it's from the igtlioImageConverter. I'll see if I can move this/any other similar copied code blocks to static functions.